full/src/main/java/apoc/load/LoadHtml.java

```diff
@@ -44,14 +44,19 @@
 
             Map<String, Object> output = new HashMap<>();
 
-            query.keySet().stream().forEach(key -> {
-                Elements elements = document.select(query.get(key));
+            query.keySet().forEach(key -> {
+                try {
+                    Elements elements = document.select(query.get(key));
 
-                output.put(key, getElements(elements, config));
+                    output.put(key, getElements(elements, config));
+
+                } catch (Exception e) {
+                    output.put(key, Collections.emptyMap());
+                }
```

Comment:
perché facciamo questo? la richiesta è di migliorare l'errore ritornato

Reply:
Da come l'ho interpretata io, bisognava ritornare una mappa vuota con le chiavi errate senza andare in eccezione come nel caso del `testQueryFailsSilently()` in cui una chiave ritorna il valore ed un'altra, andando in errore, ritorna una mappa vuota.

Reply:
allora aggiungi una properietà di config `failSilently` `true/false` (default `false`) e migliora l'output dell'eccezione per favore